lib/vtk/commands/socks/setup.rb

```diff
@@ -0,0 +1,327 @@
+            'approval before continuing. Once approved, re-run `vtk socks setup`.'
+          `#{macos? ? 'open' : 'xdg-open'} "#{url}"` if prompt.yes? 'Open access request form in GitHub?'
+        end
+
+        def configure_ssh
+          install_ssh_config
+          configure_ssh_config_with_keychain
+          ssh_config_clean_up
+          ssh_agent_add
+        end
+
+        def install_ssh_config
+          return true if ssh_config_configured?
+          return true unless prompt.yes? "----> #{pretty_ssh_config_path} missing or incomplete. Install/replace now?"
+
+          ssh_dir = File.dirname ssh_config_path
+
+          download_ssh_config unless File.exist? '/tmp/dova-devops'
+          FileUtils.mkdir_p ssh_dir
+          FileUtils.chmod 0o700, ssh_dir
+          backup_existing_ssh_config
+          FileUtils.cp '/tmp/dova-devops/ssh/config', ssh_config_path
+          FileUtils.chmod 0o600, "#{ssh_dir}/config"
+        end
+
+        def ssh_config_configured?
+          return false unless ssh_config_exists?
+
+          download_ssh_config
+          ssh_config_local = File.read ssh_config_path
+          ssh_config = File.read '/tmp/dova-devops/ssh/config'
+          ssh_config_local.include? ssh_config
+        end
+
+        def ssh_config_exists?
+          File.exist? ssh_config_path
+        end
+
+        def download_ssh_config
+          install_brew
+          log 'Downloading and checking ssh/config...'
+
+          ssh_config_clean_up
+
+          repo_url = 'https://github.com/department-of-veterans-affairs/devops.git'
+          cloned = system(
+            "git clone --quiet#{' --depth 1' if macos?} --no-checkout --filter=blob:none #{repo_url} '/tmp/dova-devops'"
+          )
+          exit 1 unless cloned
+
+          `cd /tmp/dova-devops; git checkout master -- ssh/config`
+        end
+
+        def install_brew
+          return false unless macos?
+
+          installed = !`command -v brew`.empty?
+          return true if installed
+
+          log 'Homebrew not installed. Please install and try again.'
+          `open "https://brew.sh"` if prompt.yes? 'Open https://brew.sh for installation instructions?'
+          exit 1
+        end
+
+        def ssh_config_clean_up
+          FileUtils.rm_rf '/tmp/dova-devops'
+        end
+
+        def backup_existing_ssh_config
+          return true unless File.exist? ssh_config_path
+
+          if File.exist? "#{ssh_config_path}.bak"
+            log "!!! ERROR: Could not make backup of #{pretty_ssh_config_path} as #{pretty_ssh_config_path}.bak " \
+              'exists. Aborting.'
+            exit 1
+          end
+
+          FileUtils.mv ssh_config_path, "#{ssh_config_path}.bak"
+        end
+
+        def configure_ssh_config_with_keychain
+          return unless macos?
+          return if ssh_config_configured_with_keychain?
+          return true unless prompt.yes? "----> #{pretty_ssh_config_path} missing Keychain configuration. Add now?"
```

Comment:
We should probably remove this prompt and just configure this for them automatically.